site2/docs/io-quickstart.md

```diff
@@ -1,136 +1,96 @@
-```
+    ```bash
+    curl -s http://localhost:8080/admin/v2/namespaces/public
+    ```
 
-Example outoupt:
-```shell
-["public/default","public/functions"]
-```
+    Example outoupt:
+    ```shell
+    ["public/default","public/functions"]
+    ```
 
-4. All builtin connectors should be listed as available.
+5. All builtin connectors should be listed as available.
 
-```bash
-curl -s http://localhost:8080/admin/v2/functions/connectors
-```
+    ```bash
+    curl -s http://localhost:8080/admin/v2/functions/connectors
+    ```
 
-Example output:
-```json
-[{"name":"aerospike","description":"Aerospike database sink","sinkClass":"org.apache.pulsar.io.aerospike.AerospikeStringSink"},{"name":"cassandra","description":"Writes data into Cassandra","sinkClass":"org.apache.pulsar.io.cassandra.CassandraStringSink"},{"name":"kafka","description":"Kafka source and sink connector","sourceClass":"org.apache.pulsar.io.kafka.KafkaStringSource","sinkClass":"org.apache.pulsar.io.kafka.KafkaBytesSink"},{"name":"kinesis","description":"Kinesis sink connector","sinkClass":"org.apache.pulsar.io.kinesis.KinesisSink"},{"name":"rabbitmq","description":"RabbitMQ source connector","sourceClass":"org.apache.pulsar.io.rabbitmq.RabbitMQSource"},{"name":"twitter","description":"Ingest data from Twitter firehose","sourceClass":"org.apache.pulsar.io.twitter.TwitterFireHose"}]
-```
+    Example output:
+    ```json
+    [{"name":"aerospike","description":"Aerospike database sink","sinkClass":"org.apache.pulsar.io.aerospike.AerospikeStringSink"},{"name":"cassandra","description":"Writes data into Cassandra","sinkClass":"org.apache.pulsar.io.cassandra.CassandraStringSink"},{"name":"kafka","description":"Kafka source and sink connector","sourceClass":"org.apache.pulsar.io.kafka.KafkaStringSource","sinkClass":"org.apache.pulsar.io.kafka.KafkaBytesSink"},{"name":"kinesis","description":"Kinesis sink connector","sinkClass":"org.apache.pulsar.io.kinesis.KinesisSink"},{"name":"rabbitmq","description":"RabbitMQ source connector","sourceClass":"org.apache.pulsar.io.rabbitmq.RabbitMQSource"},{"name":"twitter","description":"Ingest data from Twitter firehose","sourceClass":"org.apache.pulsar.io.twitter.TwitterFireHose"}]
+    ```
 
-If an error occurred while starting Pulsar service, you may be able to seen exception at the terminal you are running `pulsar/standalone`,
-or you can navigate the `logs` directory under the Pulsar directory to view the logs.
+    If an error occurred while starting Pulsar service, you may be able to seen exception at the terminal you are running `pulsar/standalone`,
+    or you can navigate the `logs` directory under the Pulsar directory to view the logs.
 
-## Connect Pulsar to Apache Cassandra
+# Connect Pulsar to Apache Cassandra
 
-> Make sure you have docker available at your laptop. If you don't have docker installed, you can follow the [instructions](https://docs.docker.com/docker-for-mac/install/).
+> ### Tip
```

Comment:
use four `#` - `####`. otherwise this will be rendered at the right navigation bar.

Reply:
Thanks for reminding, I've fixed.